sdk/communication/communication-chat/src/generated/src/chatApiClient.ts

```diff
@@ -50,14 +50,14 @@ export class ChatApiClient extends coreClient.ServiceClient {
       userAgentOptions: {
         userAgentPrefix
       },
-      baseUri: options.endpoint ?? options.baseUri ?? "{endpoint}"
+      endpoint: options.endpoint ?? options.baseUri ?? "{endpoint}"
```

Comment:
not sure if this is what we wanted

Reply:
Generated file so we should not manually modify this